### ISSUE TITLE
dbicdump doesn't seem to support DBD::MariaDB [rt.cpan.org #127318]

### ISSUE THREAD

**Issue by rabbiveesh** [dbsrgits/dbix-class-schema-loader#40]
Migrated from [rt.cpan.org#127318](https://rt.cpan.org/Ticket/Display.html?id=127318) (status was 'new')

Requestors:
* alex.karelas@gmail.com

From alex.karelas@gmail.com on 2018-10-09 02:18:38
:
```
I get this warning:

DBIx::Class::Storage::DBI::_warn_undetermined_driver(): This version of
DBIC does not yet seem to supply a driver for your particular RDBMS
and/or connection method ('MariaDB'). While we will attempt to continue
anyway, the results are likely to be underwhelming. Please upgrade DBIC,
and if this message does not go away, file a bugreport including the
following info:

and then underneath, more warnings and a fatal error messages, like:

DBIx::Class::Storage::DBI::sql_maker(): Your storage class
(DBIx::Class::Storage::DBI) does not set sql_limit_dialect and you have
not supplied an explicit limit_dialect in your connection_info. DBIC
will attempt to use the GenericSubQ dialect, which works on most
databases but can be (and often is) painfully slow. Please file an RT
ticket against 'DBIx::Class::Storage::DBI' at
/home/appuser/work/my_project/local/bin/dbicdump line 175

and

Cannot find primary keys for this driver:
DBIx::Class::Schema::Loader::DBI::try {...} (): DBI Exception:
DBD::MariaDB::db primary_key failed: Incorrect database name '' [for
Statement "SHOW KEYS FROM ``.`my_db`.`my_table`"] at
/home/appuser/work/my_project/local/bin/dbicdump line 175
```
